simplq/src/components/common/Nav/LeftNav.jsx

```diff
@@ -16,13 +16,22 @@ const LeftNav = ({ open, toggleClose }) => {
   return (
     <div>
       <ul className={styles['left-nav']} open={open}>
+        {page === 'home' ? (
+          <li>
+            <a tabIndex={0} role="link" onKeyDown={scrollToHowItWorks} onClick={scrollToHowItWorks}>
+              How it works
+            </a>
+          </li>
+        ) : null}
+        {page === 'admin' ? (
+          <li>
+            <a tabIndex={0} href="/">
+              My Queues
+            </a>
+          </li>
+        ) : null}
```

Comment:
Same question as mentioned in the other PR.

Reply:
My Queues is shown on admin page navbar only. And it's just a link to "/". This is a naive implementation.

I didn't want to put it on home navbar as well, because it won't do anything, and then the user would think that the link is not working.